### ISSUE TITLE
Plugin in docker hub?

### ISSUE THREAD

**Issue by jasle** [containers/docker-lvm-plugin#66]
Would it be possible to make to plugin available on the docker hub?

**Comment by jasle** [comment]
As far as I can see, there is a config.json needed, to bring a plugin to the docker hub.
https://docs.docker.com/engine/extend/config/
There is an (older) Version of the lvm-plugin in docker hub, which have an config.json in the repository:
https://github.com/nickbreen/docker-lvm-plugin

Maybe someone who is more familiar with docker, could have a look at what's to do here, to bring the lvm plugin to the docker hub?

**Comment by shishir-a412ed** [comment]
@jasle @LordMike `https://github.com/nickbreen/docker-lvm-plugin` looks like a fork of `containers/docker-lvm-plugin`, which might have support for docker-lvm-plugin as a container. We don't support that repo.
If you need that feature you are welcome to use that plugin.

We have not added the plugin as a docker image. This requires some work, and I am pretty occupied right now. So I am not sure I will get to this soon.

**Comment by olljanat** [comment]
@shishir-a412ed yep it is fork and those files looked to be working so I collected them to #96
